tools/local/check-cluster-health.sh

```diff
@@ -10,9 +10,11 @@ QC_NAME=${QC_NAME:-"qdrant-chaos-testing"}
 if [ "$QC_NAME" == "qdrant-chaos-testing" ]; then
     UPLOAD_CONTAINER_NAME="bfb-upload"
     SEARCH_CONTAINER_NAME="bfb-search"
+    POSTGRES_CLIENT_CONTAINER_NAME="postgres-client"
 elif [ "$QC_NAME" == "qdrant-chaos-testing-debug" ]; then
     UPLOAD_CONTAINER_NAME="bfb-upload-debug"
     SEARCH_CONTAINER_NAME="bfb-search-debug"
+    POSTGRES_CLIENT_CONTAINER_NAME="$POSTGRES_CLIENT_CONTAINER_NAME-debug"
```

Comment:
```suggestion
    POSTGRES_CLIENT_CONTAINER_NAME="postgres-client-debug"
```

There won't be any `POSTGRES_CLIENT_CONTAINER_NAME` defined already if it enters the `elif` block